Gemfile.lock

```diff
@@ -121,4 +121,4 @@ DEPENDENCIES
   rouge (~> 2.0.5)
 
 BUNDLED WITH
-   1.14.6
+   1.16.2
```

Comment:
What does this do?

Reply:
Seems that I used a newer version of bundled